packages/interface/src/providers/index.tsx

```diff
@@ -37,11 +38,13 @@ export const Providers = ({ children }: PropsWithChildren): JSX.Element => {
       <WagmiProvider config={config}>
         <QueryClientProvider client={queryClient}>
           <RainbowKitProvider theme={customTheme}>
-            <MaciProvider>
-              <BallotProvider>{children}</BallotProvider>
+            <RoundProvider>
+              <MaciProvider>
+                <BallotProvider>{children}</BallotProvider>
```

Comment:
It's time to add state management. There are already three providers.

Reply:
are you saying to reduce the providers to single provider, or something else?

Reply:
I'm saying about state management (redux, effector, mobx and etc)

Reply:
what would you recommend for our project?

Reply:
I prefer effector but redux is a standard. So it's up to you what to choose.

Reply:
Haven't used effector before, only used mobx and redux, let me take a look at the effector stuff and decide.

Reply:
open an issue here: #272